engine/src/main/java/org/entando/entando/web/group/model/GroupRequest.java

```diff
@@ -0,0 +1,33 @@
+package org.entando.entando.web.group.model;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * group add payload 
+ *
+ */
+public class GroupRequest extends GroupPutRequest {
+
```

Comment:
I am confused by the need for extension here. Can't be just one object?

Reply:
This is all about inheritance of  validation annotations:
in this case the put demands one required field, and the post demands the same field, with the same validation rules plus another filed...

We can have of course two independent objects

Reply:
What is specific? The payloads for PUT and POST should be identical except for whatever is the primary key field. And that is available in the URI for POST right?